### PR TITLE
cephadm: bootstrap: deploy monitoring stack by default

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
I wonder if we should be calling this the "metrics stack"?  "monitoring" doesn't seem quite right to me.

**Comment by p-se** [comment]
> I wonder if we should be calling this the "metrics stack"? "monitoring" doesn't seem quite right to me.

I think monitoring is fine. Exporteres export metrics, Prometheus processes and stores them, Alertmanager takes care of alerting and Grafana is there for visualization. Just "metrics" would be insufficient IMHO.

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -2306,6 +2306,13 @@ def command_bootstrap():
                 logger.info('Deploying %s service with default placement...' % t)
                 cli(['orch', 'apply', t])
 
+        if not args.skip_monitoring_stack:
+            logger.info('Enabling mgr prometheus module...')
+            cli(['mgr', 'module', 'enable', 'prometheus'])
+            for t in ['prometheus', 'grafana', 'node-exporter', 'alertmanager']:
+                logger.info('Deploying %s service with default placement...' % t)
+                cli(['orch', 'apply', t])
```
Comment (sebastian-philipp) [inline src/cephadm/cephadm]: Hm. Do we really want to co-locate MON+MGR and the monitoring stack on the same host by default?

cc @l-mb ?

Comment (liewegas) [reply src/cephadm/cephadm]: IMO these are small daemons.  And pretty trivial to relocate later by adjusting the placement spec.

**Comment by liewegas** [comment]
retest this please

**Pull request merged**